plugins/lead-resources/src/components/CreateCustomer.svelte

```diff
@@ -56,7 +56,7 @@
   let avatar: File | undefined
 
   function formatName (targetClass: Ref<Class<Doc>>, firstName: string, lastName: string, objectName: string): string {
-    return targetClass === contact.class.Person ? combineName(firstName, lastName) : objectName
+    return targetClass === contact.class.Person ? combineName(firstName.trim(), lastName.trim()) : objectName
```

Comment:
Probably we should check class use hierarchy.isDerived?

Reply:
targets are defined here in place, 
```
const targets = [
    client.getModel().getObject(contact.class.Person),
    client.getModel().getObject(contact.class.Organization)
  ]
```

will extend if will be required.